From 71f1603fae85f4a78c52cafdab61c76ba70d136d Mon Sep 17 00:00:00 2001
From: Paul Drews <paul.drews@intel.com>
Date: Mon, 25 Nov 2013 14:15:55 -0800
Subject: [PATCH 09/24] ACPI: Add BayTrail SoC GPIO and LPSS ACPI IDs

This adds the new ACPI ID (INT33FC) for the BayTrail GPIO
banks as seen on a BayTrail M System-On-Chip platform.  This
ACPI ID is used by the BayTrail GPIO (pinctrl) driver to
manage the Low Power Subsystem (LPSS).

Signed-off-by: Paul Drews <paul.drews@intel.com>
Acked-by: Linus Walleij <linus.walleij@linaro.org>
Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
(cherry picked from commit f6308b36c411dc5afd6a6f73e6454722bfde57b7)

This patch has been reverted in mainline, but the revert should not
longer be necessary with the x86 irq fix from Thomas Gleixner now in
mainline:

b6dae3b genirq: x86: Ensure that dynamic irq allocation does not conflict

Signed-off-by: Darren Hart <dvhart@linux.intel.com>
---
 drivers/pinctrl/pinctrl-baytrail.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/pinctrl/pinctrl-baytrail.c b/drivers/pinctrl/pinctrl-baytrail.c
index bf2b3f655469..ca998afbdb6d 100644
--- a/drivers/pinctrl/pinctrl-baytrail.c
+++ b/drivers/pinctrl/pinctrl-baytrail.c
@@ -572,6 +572,7 @@ static const struct dev_pm_ops byt_gpio_pm_ops = {
 
 static const struct acpi_device_id byt_gpio_acpi_match[] = {
 	{ "INT33B2", 0 },
+	{ "INT33FC", 0 },
 	{ }
 };
 MODULE_DEVICE_TABLE(acpi, byt_gpio_acpi_match);
-- 
1.8.1.2

